bin/truffleruby.sh

```diff
@@ -87,27 +87,33 @@ CP=""
 java_args+=("-XX:MetaspaceSize=25M")
 
 # Truffle
-binary_truffle="$root/mx.imports/binary/truffle/mxbuild"
-source_truffle="$root_parent/graal/truffle/mxbuild"
-if [ -f "$binary_truffle/dists/truffle-api.jar" ]; then # Binary Truffle suite
-    truffle="$binary_truffle"
-    graal_sdk="$(dirname "$binary_truffle")/mx.imports/binary/sdk/mxbuild/dists/graal-sdk.jar"
-    launcher_common="$(dirname "$binary_truffle")/mx.imports/binary/sdk/mxbuild/dists/launcher-common.jar"
-elif [ -f "$source_truffle/dists/truffle-api.jar" ]; then # Source Truffle suite
-    truffle="$source_truffle"
-    graal_sdk="$root_parent/graal/sdk/mxbuild/dists/graal-sdk.jar"
-    launcher_common="$root_parent/graal/sdk/mxbuild/dists/launcher-common.jar"
+binary_prefix="$root/mx.imports/binary"
+source_prefix="$root_parent"
+if [ -f "$binary_prefix/graal/truffle/mxbuild/dists/truffle-api.jar" ]; then # Binary Truffle suite
+    prefix=$binary_prefix
+elif [ -f "$source_prefix/graal/truffle/mxbuild/dists/truffle-api.jar" ]; then # Source Truffle suite
+    prefix=$source_prefix
 else
     echo "Could not find Truffle jars" 1>&2
     exit 1
 fi
-java_args+=("-Xbootclasspath/a:$truffle/dists/truffle-api.jar:$graal_sdk:$launcher_common")
-CP="$CP:$truffle/dists/truffle-nfi.jar"
+
+bootcp=(
+    "$prefix/graal/truffle/mxbuild/dists/truffle-api.jar"
+    "$prefix/graal/sdk/mxbuild/dists/graal-sdk.jar"
+    "$prefix/graal/sdk/mxbuild/dists/launcher-common.jar"
+    "$prefix/graal/tools/mxbuild/dists/chromeinspector.jar"
+    "$prefix/graal/tools/mxbuild/dists/truffle-profiler.jar"
```

Comment:
This doesn't seem correct for binary suites, there is no `graal` component and the sdk is nested under `truffle`.
Also, should we depend on the tools suite to guarantee these jars are available with binary imports?

Reply:
This should be a lot easier when using values from Mx, which I'm working. So it's probably not useful to spend time on this now.

Reply:
I'll wait for the sdk api addition then I'll investigate.